playbooks/vars/defaults.yaml

```diff
@@ -100,3 +103,15 @@ ceph_enabled: false
 # Size of the loop device that will be
 # used for Ceph (in GB).
 ceph_loop_device_size: 10
+
+sriov_enabled: false
```

Comment:
nit. is this needed?
can it be inferred from sriov_services being defined on any of the others settings